lib/agenda/every.ts

```diff
@@ -39,7 +40,7 @@ export const every = async function(this: Agenda, interval: string | number, nam
    * @param options options to run job for
    * @return array of jobs created
    */
-  const createJobs = async(interval: string | number, names: string[], data: any, options: any): Promise<Job[] | undefined> => {
+  const createJobs = async(interval: string, names: string[], data: any, options: JobOptions): Promise<Job[] | undefined> => {
```

Comment:
instead of "any" you could use "unknown" .. which forces you to do a type check  in case you want to access a property, which is way more safe.

Reply:
changed to unknown